### ISSUE TITLE
Create gesture_dataset

### ISSUE THREAD

**Issue by marcdalangin** [marcdalangin/thesis-final#12]
Create gesture dataset for hand gesture presentation

Gestures would be for the following:

- Previous Page
- Next Page
- Show Pointer
- Draw

Enhancements

- Erase Drawings

@catherenereyes @remwellll collaborate on this, use our reference as the gestures that will be using

**Comment by catherenereyes** [comment]
For Approval: 

![image](https://github.com/marcdalangin/thesis-final/assets/114873125/4b279448-41d8-4f85-baf4-5d8cec99a2b1)

**Comment by marcdalangin** [comment]
no, gamitin nyo yung gesture na sinend ko sa gc, yung ginamit from the reference natin sa video. 

1. Create a parent folder named gesture_dataset
2. Create a subfolder named for the gestures as follows:

![379543134_2921265131342864_4616645325519149160_n](https://github.com/marcdalangin/thesis-final/assets/125361057/045278c8-1d5e-4aed-a1f9-887186152a85)
Gesture for previous page - name the folder prev

![379543134_837580518046964_2764405044024629326_n](https://github.com/marcdalangin/thesis-final/assets/125361057/a37f1e7b-c5bc-408f-bd35-bc4a40e9ea15)
Gesture for next page - name the folder next

![379506468_186023691180166_7716416569837362954_n](https://github.com/marcdalangin/thesis-final/assets/125361057/38f7a996-ff6c-45dd-9bc8-c9fa15310c6f)
Gesture for show pointer - name the folder point

![379513435_220085934135548_3366476666387486156_n](https://github.com/marcdalangin/thesis-final/assets/125361057/508f974e-e157-4794-a348-b7a265a046d2)
Gesture for draw - name the folder draw

![379567973_274829865381106_3033695992483572898_n](https://github.com/marcdalangin/thesis-final/assets/125361057/f54af590-8e46-41b0-8d7e-55879e5c492b)
Gesture for undo - name the folder undo

@catherenereyes please follow ha. Thank you

**Comment by catherenereyes** [comment]
For approval: 

Note: Both left and right hand were used, as well as different comfortable resting position for the fingers for variation. 

![image](https://github.com/marcdalangin/thesis-final/assets/114873125/cff2b722-8c63-4539-a7f0-59f0bf23b540)

**Comment by marcdalangin** [comment]
Proceed. Natural, black and white backgrounds ha. Thank you

**Comment by catherenereyes** [comment]
Added the folder "gesture_dataset" along with the subfolder "draw", containing 30 images of draw gesture per background in the shared folder "THESIS_DOCS".

Note: The images are separated by folders, named by their backgrounds. Let me know if this is okay to proceed. Thank you! 

https://drive.google.com/drive/folders/1vaHaelAjiCL4pzvpWGYSizfr0JdpeBRh?usp=drive_link

**Comment by marcdalangin** [comment]
@catherenereyes - merge all photos into one folder, as instructed ko kanina, every gesture is one folder tapos doon lahat ng image magkasama yung may black, white and natural bg.

Once tapos na lahat ng gestures, lagay mo na sa repo natin, copy paste mo lang yung folder sa local copy mo ng repo the create PR, with proper commit messages.

**Comment by marcdalangin** [comment]
@catherenereyes image is too small, provide samples same as those uploaded last time, size should be 100x100

**Comment by catherenereyes** [comment]
they're 100x100, last time was 400x400, check the recent uploaded files for approval, then palitan ko yung maling folder. Thanks!

**Comment by marcdalangin** [comment]
@catherenereyes - images are approved, add the folder to the repo, name it "training_dataset". Also create a separate "validation_dataset" with a new set of images with the same gestures. Inside the parent folder validation_dataset, separate them into subfolders by gestures. Create a PR once done. Thank you!

**Comment by catherenereyes** [comment]
there are folders existing named "training_dataset" and "validation_set"

**Comment by marcdalangin** [comment]
@catherenereyes remove and replace
